### ISSUE TITLE
Date fields in date range limit are missing form labels

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#441]
Please add labels to the date range form, like so:

```
      <label class="sr-only" for="range_pub_year_tisim_begin">from year</label>

      <label class="sr-only" for="range_pub_year_tisim_end">to year</label>
```
